### PR TITLE
docs: added vimdocs

### PR DESCRIPTION
**Pull request by elianiva**
Follow up of #89. This is my first time writing vim docs so let me know what you think :)

edit: I also made the `enabled` field to true by default based on #95 so users don't have to explicitly enable it.

### CODE REVIEW

**Comment by hrsh7th** [comment]
Thank you for awesome contribution!

File: doc/nvim-compe.txt [hunk 1]
```diff
@@ -0,0 +1,352 @@
+*nvim-compe* *compe*
```
Comment (hrsh7th) [inline doc/nvim-compe.txt:1]: I prefer `compe.txt` as filename.

**Comment by hrsh7th** [comment]
LGTM.
I will merge it after filename changed.

File: doc/nvim-compe.txt (hunk 1 above)
Comment (elianiva) [reply doc/nvim-compe.txt]: done

**Comment by hrsh7th** [comment]
Thank you!

**Pull request merged**